src/cf-core/src/cf-utilities.less

```diff
@@ -246,38 +248,38 @@
 .u-w10pct  { width: 10%; }
 .u-w75pct  { width: 75%; }
 .u-w25pct  { width: 25%; }
-.u-w66pct  { width: unit((2 / 3) * 100, %); }
-.u-w33pct  { width: unit((1 / 3) * 100, %); }
+.u-w66pct  { width: unit( ( 2 / 3 ) * 100, % ); }
+.u-w33pct  { width: unit( ( 1 / 3 ) * 100, % ); }
 
 //
 // Width-specific display
 //
 
 .u-hide-on-mobile {
-    .respond-to-max(@bp-xs-max {
+    .respond-to-max( @bp-xs-max {
         display: none;
-    });
+    } );
 }
 
 .u-show-on-mobile {
     display: none;
-    .respond-to-max(@bp-xs-max {
+    .respond-to-max( @bp-xs-max {
         display: block;
-    });
+    } );
 }
 
 //
 // Small text utility
 //
 
-.u-small-text(@context: @base-font-size-px) {
-  font-size: unit(14px / @context, em);
+.u-small-text( @context: @base-font-size-px ) {
+    font-size: unit( 14px / @context, em );
 }
 
 .u-small-text {
-  .u-small-text();
+    .u-small-text();
 }
 
 small {
```

Comment:
Since this is the exact same as `u-small-text` could we just combine them to a single comma delimited rule?